docker.go

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tests
+
+import "strings"
+
+const (
+	// Docker command
+	Docker = "docker"
+
+	// Image used to run containers
+	Image = "busybox"
+)
+
+// GetContainerStatus returns the container status
+func GetContainerStatus(id string) string {
+	args := []string{"ps", "-f", "id=" + id, "--format", "{{.Status}}"}
+
+	cmd := NewCommand(Docker, args...)
+
+	exitCode := cmd.Run()
+	if exitCode != 0 {
+		return ""
+	}
+
+	stdout := cmd.Stdout.String()
+	if stdout == "" {
+		return ""
+	}
+
+	state := strings.Split(stdout, " \n\t")
+	return state[0]
+}
+
+// ContainerExist returns true if any of next cases is true:
+// - ps command shows the container
+// - the VM is running (qemu)
+// else false is returned
+func ContainerExist(id string) bool {
```

Comment:
It would be slightly more readable if this function was called `ContainerExists()`.